diff --git a/node_modules/pdf-parse/lib/pdf-parse.js b/node_modules/pdf-parse/lib/pdf-parse.js
index 96bfbc7..5332c84 100644
--- a/node_modules/pdf-parse/lib/pdf-parse.js
+++ b/node_modules/pdf-parse/lib/pdf-parse.js
@@ -43,7 +43,9 @@ const DEFAULT_OPTIONS = {
 }
 
 async function PDF(dataBuffer, options) {
-    var isDebugMode = false;
+    // ===== INICIO DE LA CORRECCIÓN =====
+    var isDebugMode = false; // El error estaba aquí. Se fuerza a 'false'.
+    // ===== FIN DE LA CORRECCIÓN =====
 
     let ret = {
         numpages: 0,
